### PR TITLE
IK 46

### PR DESCRIPTION
**Pull request by arosser**
IK-88 French language implemented into the application. All strings must now be written in English in res/values/String.xml and in French in res/values-b+fr/String.xml

### CODE REVIEW

File: client/.idea/encodings.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (handriamanantena) [inline client/.idea/encodings.xml]: why where changes added to the encodings.xml file?

File: client/app/src/main/res/values-b+fr/String.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <string name="Sign_In">Connexion</string>
+    <string name="title_activity_preference_page">page_préférences</string>
+    <string name="route_name">Nom D’Itinéraire :"</string>
+    <string name="action_settings">Paramètres</string>
+    <string name="hint_name">ex: Travaille</string>
+    <string name="Frequency">Fréquence :</string>
+    <string name="Once">Une Fois</string>
+    <string name="Repeated">Répété</string>
+    <string name="Arrival_time">Heure D’Arrivée:</string>
+    // What is rider in french?
```
Comment (handriamanantena) [inline client/app/src/main/res/values-b+fr/String.xml]: Was a proper translation found?

**Review by handriamanantena — COMMENTED** [review]
1. encodings.xml should be added to the .gitignore
2. fix comments in string file.

**Review by adrielFab — COMMENTED** [review]
There is no consistency in the naming of the strings. One string would be Sign_in and another would be hint_name. Either everything is capitalized first or everything is lowercased

File: client/app/src/main/res/values-b+fr/String.xml (hunk 2 above)
Comment (arosser) [reply client/app/src/main/res/values-b+fr/String.xml]: The string was later found not to be in any use so I will remove it altogether.

File: client/.idea/encodings.xml (hunk 1 above)
Comment (arosser) [reply client/.idea/encodings.xml]: I think they may have been added upon the creation of the french values directory.

**Pull request merged**